### ISSUE TITLE
Multiple usernames no longer work

### ISSUE THREAD

**Issue by Nobrumski** [Confuzu/CivitAI_Image_grabber#20]
Guessing they changed the API, but as of now multiple usernames no longer work. They immediately fail and report all usernames unknown. Nothing is showing in the logs either.  

Not sure if there is another way to use some sort of queue system.

**Comment by Confuzu** [comment]
i  tried the multiple usernames and it works. The multiple usernames function has nothing to  do  with  the API  its a script function.
![2342342342342323424242423423423423423](https://github.com/Confuzu/CivitAI_Image_grabber/assets/133601702/50cd8a2d-ec92-4fe8-8034-4926b3a50ecc)

**Comment by Nobrumski** [comment]
Thanks for checking, I couldn't get any multiple users to complete last night. Wonder if the API is just kicking out the same error for another reason or failing somehow on their end to find the username. I was essentially using the same settings except for HD over SD and then it would take 20 seconds or so and report all 4,3, and then 2 usernames as unknown. Other than one username, all the rest would complete fine when I ran them immediately after as single usernames. It did this every time I attempted it last night and it has been about 2 weeks since I last ran it. 

If they had just failed by themself, I would have assumed they possibly had no new images. I wonder if it is running into a timeout issue when checking the Json against what the API returns.
